lib/ui/view/dm_chat_view/dm_jump_to_viewmodel.dart

```diff
@@ -40,4 +42,174 @@ class DmJumpToViewModel extends BaseViewModel {
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'lmUnknown',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931.png',
+      text: 'noFierce',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (1).png',
+      text: 'noAbd_Salam',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (2).png',
+      text: 'Souljaunt',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (3).png',
+      text: 'noPrincess',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (4).png',
+      text: 'Nudis',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'yesUnknown',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931.png',
+      text: 'itFierce',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (1).png',
+      text: 'yesAbd_Salam',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (2).png',
+      text: 'itSouljaunt',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (3).png',
+      text: 'carsPrincess',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (4).png',
+      text: 'carsNudis',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'traUnknown',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931.png',
+      text: 'traFierce',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (1).png',
+      text: 'Abd_Salam',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (2).png',
+      text: 'traSouljaunt',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (3).png',
+      text: 'sowPrincesssssssssssssssssss',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (4).png',
+      text: 'cutNudis',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'youUnknown',
+    ),
+  ];
+
+  var recentDmsFromApi=[
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (4).png',
+      text: 'cutNudis',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'youUnknown',
+    ),
+  ];
+
+  var list=[
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (4).png',
+      text: 'cutNudis',
+    ),
+    CustomUser(
+      image: 'assets/images/Rectangle 1931 (5).png',
+      text: 'youUnknown',
+    ),
+  ];
+
+  navigateBack() {
+    navigation.back();
+  }
+
+  void _onChanged() => (value){
+
+    if(value.isEmpty){
+      list.clear();
+      list.addAll(recentDmsFromApi);
+    }else {
+      list.clear();
+      for (index in ListofDms) {
+        if (index.text.toLowerCase().contains(value.toLowerCase())) {
+          list.add(index);
+        }
+      }
+
+    }
+    notifyListeners();
+  };
+
+  //getters for the view
+  TextEditingController get controller => _controller;
```

Comment:
Use stacked_form and remove the controller from the viewmodel

Reply:
done